masonry/src/widget/widget_state.rs

```diff
@@ -148,20 +148,20 @@ impl WidgetState {
             is_explicitly_disabled: false,
             baseline_offset: 0.0,
             is_hot: false,
-            needs_layout: false,
-            needs_paint: false,
-            needs_accessibility_update: false,
-            needs_window_origin: false,
+            needs_layout: true,
+            needs_paint: true,
+            needs_accessibility_update: true,
+            needs_window_origin: true,
```

Comment:
I wonder why this was not necessary before, or was this a bug, that was just never surfaced? (xilem "classic" had this too btw. with the Changeflags)

Reply:
The default constructor in xilem classic's `ChangeFlags` would set these to true.

The way it worked in Druid and Masonry is that the default constructor set flags used by the composition root (eg `Window` for Druid, `RenderRoot` for Masonry) and then the `WidgetAdded` event set a bunch of flags to true.